ngraph/frontend/onnx_import/src/editor/editor.cpp

```diff
@@ -41,6 +42,29 @@ namespace
         {element::Type_t::u64, TensorProto_DataType::TensorProto_DataType_UINT64},
     };
 
+    inline size_t get_onnx_data_size(int32_t data_type)
+    {
+        switch (data_type)
+        {
+        case TensorProto_DataType_FLOAT: return sizeof(float);
+        case TensorProto_DataType_UINT8: return sizeof(uint8_t);
+        case TensorProto_DataType_INT8: return sizeof(int8_t);
+        case TensorProto_DataType_UINT16: return sizeof(uint16_t);
+        case TensorProto_DataType_INT16: return sizeof(int16_t);
+        case TensorProto_DataType_INT32: return sizeof(int32_t);
+        case TensorProto_DataType_INT64: return sizeof(int64_t);
+        case TensorProto_DataType_BOOL: return sizeof(char);
+        case TensorProto_DataType_FLOAT16: return 2;
+        case TensorProto_DataType_DOUBLE: return sizeof(double);
+        case TensorProto_DataType_UINT32: return sizeof(uint32_t);
+        case TensorProto_DataType_UINT64: return sizeof(uint64_t);
+        case TensorProto_DataType_COMPLEX64: return 2 * sizeof(float);
+        case TensorProto_DataType_COMPLEX128: return 2 * sizeof(double);
+
+        default: throw ngraph_error("Unhandled data type: " + data_type);
+        }
```

Comment:
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/onnx_import/src/core/tensor.hpp#L133-L157
this is the same switch, isn't it?
please use it.

Reply:
Moved to common utils.